js/globals.ts

```diff
@@ -35,13 +36,19 @@ import { libdeno } from "./libdeno";
 declare global {
   const console: consoleTypes.Console;
   const setTimeout: typeof timers.setTimeout;
+  let location: typeof LocationImpl;
```

Comment:
should only add here if it is expected to be global at build time (e.g. `main.ts` or `compiler.ts` or its imports)

Reply:
Fixed